cmd/notation/cert/add.go

```diff
@@ -28,14 +28,12 @@ func certAddCommand(opts *certAddOpts) *cobra.Command {
 			opts.path = args
 			return nil
 		},
-		Long: `Add certificates to the trust store
+		Long: `Add certificates to the trust store.`,
```

Comment:
Should we make all `Short` and `Long` consistent by removing the suffix dot `.`?

Reply:
fixed